internal/databases/mongo/oplog/oplog.go

```diff
@@ -0,0 +1,82 @@
+package oplog
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+const (
+	TimestampDelimiter = "."
+	TimestampRegexp    = `[0-9]+\` + TimestampDelimiter + `[0-9]+`
```

Comment:
Well, go way is just place the dot into literal :) But I do not think this is an issue.